File: app/src/main/java/vn/loitp/app/activity/customviews/textview/scrollnumber/ScrollNumberActivity.java
package vn.loitp.app.activity.customviews.textview.scrollnumber;

import android.app.Activity;
import android.os.Bundle;
import vn.loitp.core.base.BaseActivity;
import loitp.basemaster.R;
import vn.loitp.views.textview.scrollnumber.lib.MultiScrollNumber;

public class ScrollNumberActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        MultiScrollNumber scrollNumber = (MultiScrollNumber) findViewById(R.id.scroll_number);

        scrollNumber.setTextColors(new int[]{R.color.red01, R.color.orange01, R.color.blue01, R.color.green01, R.color.purple01});
        //scrollNumber.setTextSize(64);

        //scrollNumber.setNumber(64, 2048);
        //scrollNumber.setInterpolator(new DecelerateInterpolator());


        scrollNumber.setScrollVelocity(20);
        scrollNumber.setNumber(20.48);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_scroll_number;
    }

}

File: app/src/main/java/vn/loitp/app/activity/customviews/textview/scrollNumber/ScrollNumberActivity.kt
package vn.loitp.app.activity.customviews.textview.scrollNumber

import android.os.Bundle
import com.loitpcore.annotation.IsFullScreen
import com.loitpcore.annotation.LogTag
import com.loitpcore.core.base.BaseFontActivity
import com.loitpcore.core.utilities.LUIUtil
import kotlinx.android.synthetic.main.activity_text_view_scroll_number.*
import vn.loitp.app.R

@LogTag("ScrollNumberActivity")
@IsFullScreen(false)
class ScrollNumberActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_text_view_scroll_number
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
    }

    private fun setupViews() {
        lActionBar.apply {
            LUIUtil.setSafeOnClickListenerElastic(
                view = this.ivIconLeft,
                runnable = {
                    onBaseBackPressed()
                }
            )
            this.ivIconRight?.setImageResource(R.color.transparent)
            this.tvTitle?.text = ScrollNumberActivity::class.java.simpleName
        }
        multiScrollNumber.setTextColors(
            intArrayOf(
                R.color.red50,
                R.color.orange,
                R.color.blue,
                R.color.green,
                R.color.purple
            )
        )
        // scrollNumber.setTextSize(64)

        // scrollNumber.setNumber(64, 2048)
        // scrollNumber.setInterpolator(new DecelerateInterpolator())

        multiScrollNumber.setScrollVelocity(20)
        multiScrollNumber.setNumber(20.48)
    }
}
